### DISCUSSION TITLE
Any Success Registering Hyperchhalk in Instructure Canvas?

### DISCUSSION THREAD

**Discussion by anotherspot** [Hyperchalk/Hyperchalk#7] [Q&A]
I have attempted to register the OneOffRegistrationLink in Canvas.

Canvas returns the following message when adding the LTI 1.3 App using the LTI 2 configuration URL option.

There was an error while registering you application: No configuration endpoint was found in the parameters. Are you trying to register a legacy LTI consumer? This app only supports LTI 1.3 Advantage.

Has anyone had some success?

**Comment by onemenzel** [comment]
That is strange indeed as HC actually only supports LTI 1.3. Unfortunately, I don't have any access to the official LTI test suite, which is why I can only fix LTI support with open source LMSes that I can access via a debugger.

I'd appreciate it, too if anyone could provide us with information on this issue.

**Reply by anotherspot** [reply]
Can you test using the Canvas open source github repo?
https://github.com/instructure/canvas-lms
